app/controllers/concerns/authentication_helper.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+module AuthenticationHelper
+  def login(user)
+    reset_session
```

Comment:
is it a built in method? if not then where it is defined?

Reply:
Bhaiya, **reset_session** is built in methods in rails.